src/datadogConnector/utils/canInitDatadog.test.ts

```diff
@@ -0,0 +1,52 @@
+import { before } from 'node:test'
```

Comment:
Same as above; I think it's probably best to stick with the Jest globals? 🤔 At least, I don't know if this is doing what we want it to. I _do_ know that Jest does some _funny_ things under the hood with parallelization via spawning child processes, basically-kinda-cloning all the in-memory variables for reach child process, and running its own lifecycle functions, so I'm not sure how this interacts with all that.